packages/client/src/v2-events/features/events/actions/declare/Review.tsx

```diff
@@ -88,6 +80,171 @@ const messages = defineMessages({
+          'The description for registration action when form is complete'
+      },
+      onConfirm: registerMessages.onConfirm
+    },
+    validate: {
+      title: validateMessages.title,
+      description: {
+        id: 'v2.review.validate.description.complete',
+        defaultMessage:
+          'The informant will receive an email with a registration number that they can use to collect the certificate',
+        description: 'The description for validate action when form is complete'
+      },
+      onConfirm: validateMessages.onConfirm
+    },
+    declare: {
+      title: {
+        id: 'v2.review.declare.title.complete',
+        defaultMessage: 'Declaration complete',
+        description:
+          'The title shown when reviewing an incomplete record to declare'
+      },
+      description: {
+        id: 'v2.review.declare.description.complete',
+        defaultMessage:
+          'The informant will receive an email with a registration number that they can use to collect the certificate',
+        description: 'The description for declare action when form is complete'
+      },
+      onConfirm: declareMessages.onConfirm
+    }
+  },
+  incomplete: {
+    register: {
+      title: registerMessages.title,
+      description: {
+        id: 'v2.reviewAction.register.description.incomplete',
+        defaultMessage: 'Please add mandatory information before registering',
+        description:
+          'The description for registration action when form is incomplete'
+      },
+      onConfirm: registerMessages.onConfirm
+    },
+    validate: {
+      title: validateMessages.title,
+      description: {
+        id: 'v2.review.validate.description.incomplete',
+        defaultMessage:
+          'Please add mandatory information before sending for approval',
+        description: 'The description for validate action when form is complete'
+      },
+      onConfirm: validateMessages.onConfirm
+    },
+    declare: {
+      title: {
+        id: 'v2.review.declare.title.incomplete',
+        defaultMessage: 'Declaration incomplete',
+        description:
+          'The title shown when reviewing an incomplete record to declare'
+      },
+      description: {
+        id: 'v2.review.declare.description.complete',
+        defaultMessage:
+          'The informant will receive an email with a tracking ID that they can use to provide the additional mandatory information required for registration',
+        description: 'The description for declare action when form is complete'
+      },
+      onConfirm: declareMessages.onConfirm
+    }
+  }
+}
+function getReviewActionConfig({
+  formConfig,
+  form,
+  metadata,
+  scopes
+}: {
+  formConfig: FormConfig
+  form: ActionFormData
+  metadata?: ActionFormData
+  scopes?: Scope[]
+}) {
+  const incomplete = validationErrorsInActionFormExist(
+    formConfig,
+    form,
+    metadata
+  )
+
+  const isDisabled = incomplete
+    ? !!scopes?.includes(SCOPES.RECORD_SUBMIT_INCOMPLETE)
```

Comment:
```suggestion
    ? !scopes?.includes(SCOPES.RECORD_SUBMIT_INCOMPLETE)
```
Should be disabled when inComplete && scopes does not include `SCOPES.RECORD_SUBMIT_INCOMPLETE`. Or is there something I am missing?

Reply:
Thanks, you are right 🙏